babel/board.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 import * as _ from 'lodash';
+let SortedSet = require('collections/sorted-set');
```

Comment:
Node v5 should come with `Set` objects already. Is it just as easy as creating a set, and then sorting it?